components/ItemGrid/ItemGridOptions.bs

```diff
@@ -14,7 +14,7 @@ sub init()
     background.blendColor = ColorPalette.LIGHTGREY
 
     m.buttons = m.top.findNode("buttons")
-    m.buttons.buttons = [tr("View"), tr("Sort"), tr("Filter")]
+    m.buttons.buttons = [tr("TAB_View"), tr("TAB_Sort"), tr("TAB_Filter")]
```

Comment:
These strings don't match the casing in the translation file, so they are not translated.

![WIN_20250708_08_20_25_Pro](https://github.com/user-attachments/assets/2c3daace-66d2-4a7c-9b9a-fc13d5d90bb6)